python/dgl/data/gindt.py

```diff
@@ -79,58 +78,53 @@ def __init__(self, name, self_loop, degree_as_nlabel=False):
         self.degree_as_nlabel = degree_as_nlabel
         self.nattrs_flag = False
         self.nlabels_flag = False
-        self.verbosity = False
 
-        # calc all values
-        self._load()
+        super(GINDataset, self).__init__(name=name, url=gin_url,
+                                         raw_dir=raw_dir, force_reload=force_reload, verbose=verbose)
+
+    @property
+    def raw_path(self):
+        return os.path.join(self.raw_dir, 'GINDataset')
+
+    def download(self):
+        r""" Automatically download data and extract it.
+        """
+        zip_file_path = os.path.join(self.raw_dir, 'GINDataset.zip')
+        download(self.url, path=zip_file_path)
+        extract_archive(zip_file_path, self.raw_path)
 
     def __len__(self):
         """Return the number of graphs in the dataset."""
         return len(self.graphs)
 
     def __getitem__(self, idx):
         """Get the i^th sample.
-
         Paramters
         ---------
         idx : int
             The sample index.
-
         Returns
         -------
-        (dgl.DGLGraph, int)
+        (dgl.Graph, int)
```

Comment:
@jermainewang Is this DGLGraph or Graph?

Reply:
```
>>> import dgl
Using backend: pytorch
>>> g = dgl.graph([])
>>> g
Graph(num_nodes=0, num_edges=0,
      ndata_schemes={}
      edata_schemes={})
```

Reply:
I check with mufei, this maybe some inconsistency with DGLGraph and DGLHeteroGraph. 
Let's left it there and fix it later.